Cargo.toml

```diff
@@ -7,7 +7,7 @@ repository = "https://github.com/volta-cli/volta"
 edition = "2018"
 
 [features]
-universal-docs = ["volta-core/universal-docs"]
+cross-platform-docs = ["volta-core/cross-platform-docs"]
```

Comment:
The `universal-docs` feature flag was kind of non-self-explanatory, so I rename it to `cross-platform-docs`, which I think makes it a little easier to guess the meaning of. TLDR it's a feature flag you enable to do a single build of the API docs that will include both Unix- and Windows-specific modules.